docs/batch.rst

```diff
@@ -195,66 +195,38 @@ If you have thousands of files to work with, contact the author.
 Consulting work related to OCRmyPDF helps fund this open source project
 and all inquiries are appreciated.
 
-Hot (watched) folders
-=====================
-
-To set up a "hot folder" that will trigger OCR for every file inserted,
-use a program like Python
-`watchdog <https://pypi.python.org/pypi/watchdog>`__ (supports all major
-OS).
-
-One could then configure a scanner to automatically place scanned files
-in a hot folder, so that they will be queued for OCR and copied to the
-destination.
-
-.. code-block:: bash
-
-   pip install watchdog
+Watched folders
```

Comment:
Not everyone wants to use Docker so let's retain the existing instructions for using watchdog/watchmedo directly.

It can/should also work to run watcher.py directly rather than in Docker.